src/providers/ProviderFactory.ts

```diff
@@ -125,11 +125,7 @@ export class ProviderFactory {
     setAccountProvider(dappProvider);
 
     const shouldClearInitiatedLogins = (
-      [
-        ProviderTypeEnum.crossWindow,
-        ProviderTypeEnum.metamask,
-        ProviderTypeEnum.passkey
-      ] as readonly ProviderType[]
+      [ProviderTypeEnum.crossWindow] as readonly ProviderType[]
     ).includes(type);
```

Comment:
Does this still need to be an array and check against the type with `.includes()` since there's just one item now?